.ci/pipeline/docs.yml

```diff
@@ -16,26 +16,47 @@ pr:
     - doc
     - examples
 
+variables:
+  - name: python.version
+    value: '3.7'
```

Comment:
we use 3.8 as default in other places - should we use 3.7 here?

Reply:
For the conda now we use python37 everywhere, due to the fact that the daal is the freshest in the Intel channel now.